package/MDAnalysis/coordinates/base.py

```diff
@@ -1132,12 +1137,7 @@ def _read_next_timestep(self, ts=None):  # pragma: no cover
 
     def __iter__(self):
         self._reopen()
```

Comment:
So I've just noticed that for iteration we reopen the file, and file handle aren't cheap (see @kain88-de in #239) so we should probably have `rewind` just seek to the start of the file rather than reopen the file.